lib/stdlib/src/sets.erl

```diff
@@ -129,20 +129,14 @@ is_element(E, S) ->
 add_element(E, S0) ->
     Slot = get_slot(S0, E),
     Bkt = get_bucket(S0, Slot),
-    case add_bkt_el(E, Bkt, Bkt) of
-        not_changed ->
+    case lists:member(E, Bkt) of
+        true ->
             S0;
-        NewBkt ->
-            S1 = update_bucket(S0, Slot, NewBkt),
+        _ ->
```

Comment:
Please don't use '_' to match a single, known value. Make it explicit like this:

    false ->